canonical-web-and-design/ubuntu.com
# Add legal shortlink for the ip policy

## Done

- Add shortlink from /legal/IP to /legal/intellectual-property-policy

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/legal/IP
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- See that it resolves to http://0.0.0.0:8001/legal/intellectual-property-policy
